spec/support/shared/examples/payload_cached_size_is_consistent.rb

```diff
@@ -102,6 +102,7 @@
       'RC4PASSWORD' => 'Metasploit',
       'DNSZONE' => 'corelan.eu',
       'PEXEC' => '/bin/sh',
+      'HttpUserAgent' => 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:94.0) Gecko/20100101 Firefox/94.0',
```

Comment:
Are these set strings so that the payload cached size always remains the same?

Reply:
Yeah, that was the intent (more words above)